### PR TITLE
Update release information for 2.2 doc archive

### PR DESCRIPTION
**Pull request by leslietilling**
## Purpose of this pull request

This pull request (PR) updates the "About this Release" section that provides information and doc links for prior M2 versions. The 2.2 online content will be removed from our site after the 2.4 release.  This change should go to master first and then be synched to the 2.4.0 branch prior to GA. 

If Magento Commerce, please specify:

- [ ] Commerce only?
- [ ] Commerce with B2B?

## Affected documentation pages

- https://docs.magento.com/user-guide/magento/about-this-release.html

### CODE REVIEW

File: src/magento/about-this-release.md [hunk 1]
```diff
@@ -29,31 +29,25 @@ If you need documentation for an earlier Magento release, the following resource
...
-**Release Information:** [Magento 2.2.x Release Information][16]{:target="_blank"}
-
-**Online User Guide:**
-
-- {:.ce-only}[Magento Open Source 2.2 User Guide][13]{:target="_blank"}
-- {:.ee-only}[Magento Commerce 2.2 User Guide][14]{:target="_blank"}
-- {:.b2b-only}[Magento Commerce for B2B 2.2 User Guide][15]{:target="_blank"}
-
-**PDF User Guide:**
+Magento 2.2.11 was the final 2.2.x release. After April 2020, Magento 2.2.x stopped receiving security patches, quality fixes, or documentation updates. To maintain your site's performance, security, and PCI compliance, you should upgrade to the latest version of Magento. User documentation remains available in PDF form:
```
Comment (hickskenh) [inline src/magento/about-this-release.md:32]:
```suggestion
Magento 2.2.11 was the final 2.2.x release. After April 2020, Magento 2.2.x stopped receiving security patches, quality fixes, and documentation updates. To maintain your site's performance, security, and PCI compliance, you should upgrade to the latest version of Magento. User documentation is no longer updated but remains available in PDF form:
```

File: src/magento/about-this-release.md [hunk 2]
```diff
@@ -29,31 +29,25 @@ If you need documentation for an earlier Magento release, the following resource
...
 
 - {:.ce-only}[Magento Open Source 2.2 User Guide][6]{:target="_blank"}
 - {:.ee-only}[Magento Commerce 2.2 User Guide][7]{:target="_blank"}
 - {:.b2b-only}[Magento Commerce for B2B 2.2 User Guide][8]{:target="_blank"}
 
 ### 2.1
 
-Magento 2.1.18 was the final 2.1.x release. After June 2019, Magento 2.1.x stopped receiving security patches, quality fixes, or documentation updates. To maintain your site's performance, security, and PCI compliance, you should upgrade to the latest version of Magento.
+Magento 2.1.18 was the final 2.1.x release. After June 2019, Magento 2.1.x stopped receiving security patches, quality fixes, or documentation updates. To maintain your site's performance, security, and PCI compliance, you should upgrade to the latest version of Magento. User documentation remains available in PDF form:
```
Comment (hickskenh) [inline src/magento/about-this-release.md:40]:
```suggestion
Magento 2.1.18 was the final 2.1.x release. After June 2019, Magento 2.1.x stopped receiving security patches, quality fixes, and documentation updates. To maintain your site's performance, security, and PCI compliance, you should upgrade to the latest version of Magento. User documentation is no longer updated but remains available in PDF form:
```

File: src/magento/about-this-release.md [hunk 3]
```diff
@@ -29,31 +29,25 @@ If you need documentation for an earlier Magento release, the following resource
...
 
-The [Magento Community 2.1 User Guide][5]{:target="_blank"} and [Magento Enterprise 2.1 User Guide][4]{:target="_blank"} and remain available in PDF form.
+- {:.ce-only}[Magento Community 2.1 User Guide][5]{:target="_blank"}
+- {:.ee-only}[Magento Enterprise 2.1 User Guide][4]{:target="_blank"}
 
 ### 2.0
 
-Magento 2.0.18 was the final 2.0.x release. After March 2018, Magento 2.0.x stopped receiving security patches, quality fixes, and documentation updates. To maintain your site's performance, security, and PCI compliance, you should upgrade to the latest version of Magento.
+Magento 2.0.18 was the final 2.0.x release. After March 2018, Magento 2.0.x stopped receiving security patches, quality fixes, and documentation updates. To maintain your site's performance, security, and PCI compliance, you should upgrade to the latest version of Magento. User documentation remains available in PDF form:
```
Comment (hickskenh) [inline src/magento/about-this-release.md:47]:
```suggestion
Magento 2.0.18 was the final 2.0.x release. After March 2018, Magento 2.0.x stopped receiving security patches, quality fixes, and documentation updates. To maintain your site's performance, security, and PCI compliance, you should upgrade to the latest version of Magento. User documentation is no longer updated but remains available in PDF form:
```

**Comment by leslietilling** [comment]
running tests

**Comment by leslietilling** [comment]
running tests

**Pull request merged**